testing.go

```diff
@@ -133,3 +129,57 @@ func createStubHTTPClient(t *testing.T, fixtures []string) Client {
 func createStubHTTPClientSingle(t *testing.T, fixture string) Client {
 	return createStubHTTPClient(t, []string{fixture})
 }
+
+func sortMethodCall(methodCall MethodCall) {
+	for _, param := range methodCall.Params {
+		value := param.Value
+		if len(value.Struct.Members) > 0 {
+			sortStruct(value.Struct)
+		}
+		// Arrays should already be correctly handled since they are insertion ordered.
+	}
+}
+
+func sortStruct(xmlrpcStruct Struct) {
+	sort.Slice(xmlrpcStruct.Members, func(i, j int) bool {
+		return xmlrpcStruct.Members[i].Name < xmlrpcStruct.Members[j].Name
+	})
+	for _, value := range xmlrpcStruct.Members {
+		if len(value.StructValue.Struct.Members) > 0 {
+			sortStruct(value.StructValue.Struct)
+		}
+	}
+}
+
+type Member struct {
+	Name        string      `xml:"name"`
+	StructValue XMLRPCValue `xml:"value"`
+}
+
+type Struct struct {
```

Comment:
I think Struct is a very poorly chosen name. Something that differentiates it more from the builtin struct would be good, even xmlrpcStruct, or xmlrpcObject or -Response

Reply:
I have renamed them to the same schema as the Value Struct.